### ISSUE TITLE
Gatekeeper Three, intro/outro text

### ISSUE THREAD

**Issue by t-gm** [OpenZeppelin/ethernaut#613]
Hi,

after solving this level I have to say that either the texts in the intro / outro do not reflect the level, or there was a (way simpler?) solution to solving the level.

I am specifically talking about

Intro:
`Refresh how storage works in Ethereum.`

Outro: links to "Ethereum Solidity: Memory vs Storage & How to initialize an array inside a struct" and the link to "getStorageAt" function for web3js.

None of these have any relevance to the challenge? Or am I missing something?

**Comment by t-gm** [comment]
My point was, you can just use `createTrick` and `getAllowance(block.timestamp)` in the same transaction so you don't need to read via getStorageAt (that is why I was confused as to why the outro was hinting at that).

**Comment by 0xKMG** [comment]
Agree, if the intro doesn't mention storage I'll probably use the method you've mentioned to solve. 

Well at least it is one of the possible ways to solve so I am gonna look pass that, what's the deal with `Recall return values of low-level functions` ? That I'm really confused.

**Comment by GianfrancoBazzani** [comment]
The game has multiple solutions and that is the objective, I proceed to close this issue.
